MinPlatformPkg/Test/TestPointStubDxe/TestPointStubDxeSmm.inf

```diff
@@ -12,10 +12,10 @@
   FILE_GUID                      = C034A2CB-1190-4638-9DDF-6F297824741E
   VERSION_STRING                 = 1.0
   MODULE_TYPE                    = DXE_DRIVER
-  ENTRY_POINT                    = TestPointStubDxeEntryPoint
+  ENTRY_POINT                    = TestPointStubDxeSmmEntryPoint
```

Comment:
You'll probably need to rework the tweaks currently in TestPointStubDxeSmm based on evaluating how to consolidate changes down to what is needed for the new code.

Reply:
Consolidated the Standalone MM and SMM files.